### DISCUSSION TITLE
[Self Hosted][Auth Provider] What means Microsoft as auth provider

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#259] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11.0.2

### Description

Hi guys,
reading the [documentation for "_Self Hosted_" infrastructures](https://docs.mia-platform.eu/docs/self_hosted/self_hosted_requirements#software-and-hardware-prerequisites), I found the following table:
<img width="908" alt="Screenshot 2023-05-23 at 12 02 46" src="https://github.com/mia-platform/community/assets/28152586/24351ccb-3051-40e2-b52d-a54f55573b04">

The table contains "Microsoft" as auth provider, but it's very generic. You are talking about "Active Directory" or some other Microsoft login tool like "Azure AD"?

Thanks.

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! It is the Microsoft social login. We recently added also the Microsoft Azure AD B2C, which we should add here. For more information, you could see [here](https://docs.mia-platform.eu/docs/runtime_suite/authentication-service/configuration#how-to-configure-providers).

What do you think should be better in the table? Both `Microsoft Social Login` and `Microsoft's Azure AD B2C` in two different rows?

**Reply by frengooo** [reply]
Hi @davidebianchi, thanks for your reply.
I think both rows is a good option, otherwise we should add another column to specify the integration types (for example), but probably it's better in the future (now only the `Microsoft` row needs it).

**Reply by davidebianchi** [reply]
Perfect! @albertotessarotto can we add it to the list?
